scripts/start_mcp.sh

```
@@ -0,0 +1,28 @@
+LOGFILE="$3"
+MCP_PORT="$4"
+CONFIG_FILE="${5}"
+TRANSPORT="http"
+
+if [[ ! -f "${EXE}" ]]; then
+  echo "The MCP server executable ${EXE} does not exist"
+  exit 2
+fi
+
+if [[ "${CONFIG_FILE}" != "" ]]; then
+  "${EXE}" --transport ${TRANSPORT} --port ${MCP_PORT} --config "${CONFIG_FILE}" >> "${LOGFILE}" 2>&1 &
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)